docs/5.0/kubernetes-access.md

```diff
@@ -5,61 +5,231 @@ description: How to set up and configure Teleport for Kubernetes access with SSO
+as [LoadBalancer](https://github.com/gravitational/teleport/blob/master/examples/chart/teleport/values.yaml#L197)
+
+## Connect Kubernetes cluster to an external Teleport Access Plane
+Users can connect multiple Kubernetes clusters to an external Teleport Access Plane
+this helps consolidate SSO, RBAC and kubectl Auditing.
+
+Kubernetes can connect to the external Teleport cluster using two possible options:
+
+1. Using **Trusted Clusters** connecting to a Teleport root cluster. This method leverages
+[Trusted Clusters](trustedclusters.md) feature to connect a Teleport auth & proxy
+back to a root cluster. This can be useful when customers want to isolate environments.
+ _e.g. A MSP provide is managing customer infrastructure_
+2. Using **Kube Agent** connecting to a root Teleport Access Plane.  This method
+leverages Teleports use of tunnels and is a lightweight way to quickly setup Teleport.
+All Audit log activity and access will happen on the root. This can be useful for managing
+multiple environments inside the same company. _e.g. providing access to staging and production
+clusters_
+
+![teleport-outside-kubernetes](img/kubernetes-eks-gcp.svg)
+
+=== "Using Kube Agent"
+
+    Prepare Teleport Root<br>
+    The  Teleport Cluster should be setup following our standard config. For
+    clients to connect need an invite token is set for the `kube` service and
+    proxy_addr has `kube_listen_addr` set.
+
+    ```yaml
+    # Example Snippet for the teleport.yaml for Teleport root cluster
+    #...
+    auth_service:
+    # Example Static Kube invite token. Note V1 only supports static
+    # invite tokens.
+      tokens:
+      - kube:866c6c114724a0fa4d4d73216afd99fb1a2d6bfde8e13a19
+    #...
+    proxy_service:
+      public_addr: proxy.example.com:3080
+      kube_listen_addr: 0.0.0.0:3026
+    ```
+
+    Setup Teleport Helm chart repository.
+
+    ```bash
+    helm repo add teleport https://charts.releases.teleport.dev
+    ```
+
+    | Things to set | Description |
+    |-|-|
+    | `proxyAddr` | The Address of the Teleport Root Service, using the proxy listening port |
+    | `authToken` | A static `kube` invite token |
+    | `kubeClusterName` | Kubernetes Cluster name (there is no easy way to automatically detect the name from the environment) |
+
+    Install the Helm Chart.
+
+    ```bash
+    helm install teleport-kube-agent teleport/teleport-kube-agent \
+      --namespace teleport \
+      --create-namespace \
+      --set proxyAddr=proxy.example.com:3080 \
+      --set authToken=$JOIN_TOKEN \
+      --set kubeClusterName=$KUBERNETES_CLUSTER_NAME
+    ```
+
+=== "Using Trusted Cluster: Teleport Enterprise Example"
```

Comment:
Should this explain how to set up the root cluster, same as we do above for kube agent?